models/reviews_model.js

```diff
@@ -77,3 +77,26 @@ exports.insertComment = (review_id, newComment) => {
       return comment.rows[0];
     });
 };
+
+exports.updateReview = (review_id, updatedReview) => {
+  if (!updatedReview.hasOwnProperty("inc_votes")) {
+    return Promise.reject({
+      status: 400,
+      msg: "Bad Request",
+    });
+  }
+  const { inc_votes } = updatedReview;
+  if (isNaN(review_id) || isNaN(inc_votes)) {
+    return Promise.reject({
+      status: 400,
+      msg: "Bad Request",
+    });
+  }
+  return checkExists("reviews", "review_id", review_id)
+    .then(() => {
+      return db.query(`UPDATE reviews SET votes = votes + ${inc_votes} WHERE review_id = ${review_id} RETURNING *`);
```

Comment:
I'm assuming your checkExists function rules out any values regarding sql injection - it still makes me feel a little strange here seeing values interpolated directly. I appreciate however you've effectively greenlisted the values due to `isNaN` above